### ISSUE TITLE
use Zenodo’s DOI versioning for blog posts

### ISSUE THREAD

**Issue by BatoolMM** [rstudio/distill#262]
I was wondering if it's possible to use Zenodo API to create a DOI for each blog post and add the DOI to the citation 🤔.
Similar work was done [here](
https://o2r.info/2019/02/24/archiving-jekyll-zenodo/) 🙏 but with Jekyll.

**Comment by jjallaire** [comment]
I agree it would be very cool to automate all of this, but it might take quite a bit of doing to make it seamless and foolproof. Not something we have time to do right now, hopefully in the meantime it should be easy enough to just render a PDF version of the post and upload it to get your DOI.
